flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/connectors/hive/HiveTableSourceITCase.java

```diff
@@ -447,6 +447,51 @@ public void testProjectionPushDown() throws Exception {
         }
     }
 
+    @Test
+    public void testReadParquetNestedPushdown() throws Exception {
+        batchTableEnv.executeSql(
+                "create table parquet_complex_type_test("
+                        + "a array<struct<a1:int>>, "
+                        + "m map<int,string>, "
+                        + "s struct<f1:int,`f2.q1`:bigint,f3:DECIMAL(5,4), f4:struct<f5:string>>"
```

Comment:
Can we define the `f4` as `f4:struct<f5:string, f6: string>`? then we just select the `f5` field.